code/game/objects/structures/teleporter.dm

```diff
@@ -15,27 +15,39 @@
 		/obj/machinery/nuclearbomb
 	)
 
+/obj/machinery/deployable/teleporter/examine(mob/user)
+	. = ..()
+	var/obj/item/teleporter_kit/kit = internal_item
+	if(!kit.cell)
+		. += "It is currently lacking a power cell."
+	if(kit.linked_teleporter)
+		. += "It is currently linked to Teleporter #[kit.linked_teleporter.tele_tag] at [get_area(kit.linked_teleporter)]"
+	else
+		. += "It is not linked to any other teleporter."
+
+
 /obj/machinery/deployable/teleporter/Initialize()
 	. = ..()
 	SSminimaps.add_marker(src, z, MINIMAP_FLAG_MARINE, "teleporter")
 
+
 /obj/machinery/deployable/teleporter/attack_hand(mob/living/user)
 	. = ..()
 	var/obj/item/teleporter_kit/kit = internal_item
 	if(!istype(kit))
 		CRASH("A teleporter didn't have an internal item, or it was of the wrong type.")
 
 	if (!powered() && (!kit.cell || kit.cell.charge < TELEPORTING_COST))
-		to_chat(user, span_warning("A red light flashes on the [src]. It seems it doesn't have enough power."))
+		to_chat(user, span_warning("A red light flashes on [src]. It seems it doesn't have enough power."))
```

Comment:
This is what `\proper` and `\improper` should solve.
Instead of removing `the` add `\the`

Reply:
Done. Thank you.